quickwit/quickwit-proto/protos/quickwit/search_api.proto

```diff
@@ -240,8 +240,8 @@ message LeafSearchResponse {
   // num_attempted_splits = num_successful_splits + num_failed_splits.
   uint64 num_attempted_splits = 4;
 
-  // json serialized intermediate aggregation_result.
-  optional string intermediate_aggregation_result = 5;
+  // postcard serialized intermediate aggregation_result.
+  optional bytes intermediate_aggregation_result = 5;
```

Comment:
```suggestion
  // json serialized intermediate aggregation_result.
  reserved 5;

  // postcard serialized intermediate aggregation_result.
  optional bytes intermediate_aggregation_result = 6;
  
  
    // deprecated tag field
  reserved 8;
```

Reply:
Do we need this? We break compatibility anyway